python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -1259,7 +1241,8 @@ def test_dataframe_concat_different_numerical_columns(dtype1, dtype2):
     else:
         pres = pd.concat([df1, df2])
         gres = cudf.concat([cudf.from_pandas(df1), cudf.from_pandas(df2)])
-        assert_eq(cudf.from_pandas(pres), gres)
+        # Pandas 1.3.2+ returns mixed `object` dtype result.
+        assert_eq(cudf.from_pandas(pres.astype(gres.dtypes)), gres)
```

Comment:
I think in situations like this, it's cleaner to use `assert_eq(expect, got, check_dtypes=False)` -- or does that not achieve the same thing?

Reply:
Yup, changed this.